src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -38,6 +38,7 @@ export interface StatementGroupComposition {
 export interface ListStatementGroupsOptions {
     page?: number;
     perPage?: number;
+    filter?: string;
```

Comment:
Can you add the JSdoc please? :smile:

Reply:
Do we have any guidelines for the JSDoc? Like what do I need to describe, how much details, do I need example, etc

Reply:
We don't have written guidelines I think :thinking:  @gdostie ? :eyes: 
But here's an example ---> https://github.com/coveo/platform-client/blob/master/src/resources/Catalogs/CatalogInterfaces.ts

I don't think example are necessary since we want documentation for the properties only.

Reply:
There is no guideline yet, but just copy-pasting what's already in swagger doc is a good start. Just remember that the goal is so that a user of the lib understands what he is doing.